src/pages/[platform]/build-a-backend/data/enable-logging/index.mdx

```diff
@@ -0,0 +1,95 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Enable logging for your data resource',
+  description: 'Enable logging for your data resource',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+You can enable logging to debug your GraphQL API using CloudWatch logs.
+
+## Default Logging
+
+Default logging can be enabled by setting the `logging` property to `true` in the call to `defineData`. For example:
+
+```ts title="amplify/data/resource.ts"
+export const data = defineData({
+  some_schema,
+  some_authorizationModes,
```

Comment:
```suggestion
  // ...
```

I think for this we can either show a full example with stubs for the property values or a comment that denotes some information precedes the logging prop. This example is using the property shorthand syntax assigning the variable `some_schema` to the property `some_schema`, i.e.

```ts
const some_schema = a.schema({})

defineData({
  some_schema: some_schema,
})
```